package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@daostack/subgraph-experimental",
-  "version": "0.1.1-rc.8-v10",
+  "name": "@dorgtech/subgraph-experimental",
```

Comment:
Please remove this change...

Reply:
Also, why not v11?

Reply:
Because we were deploying it multiple time in order to release the `test_env` and test arc, which is also done and tested :-)